sdk/personalizer/Azure.AI.Personalizer/src/Models/DecisionContextDocument.cs

```diff
@@ -11,76 +11,52 @@ public class DecisionContextDocument
     {
         /// <summary> Initializes a new instance of DecisionContextDocument. </summary>
         /// <param name="id"> Id of the decision context document </param>
-        /// <param name="json"> The json features </param>
+        /// <param name="actionFeatureJsons"> The json list of action features </param>
         /// <param name="slotId"> The slot Id </param>
-        /// <param name="slotJson"> The slot json features </param>
-        public DecisionContextDocument(string id, List<string> json, string slotId, List<string> slotJson)
+        /// <param name="slotFeatureJsons"> The json list of slot features </param>
+        public DecisionContextDocument(string id, List<string> actionFeatureJsons, string slotId, List<string> slotFeatureJsons)
         {
-            ID = id;
-            JSON = json;
+            Id = id;
+            ActionFeatureJsons = actionFeatureJsons;
```

Comment:
[nit] it is one json with one or more action features, this reads as multiple jsons for action features

same thing goes for slot below

Reply:
They are actually a list of json strings. See the following screenshot:
![image](https://user-images.githubusercontent.com/90279783/153963016-ded213ef-f614-494b-8ce6-840fb4e4dd7c.png)